### PR TITLE
mark-up en-dashes properly, typos

### PR DESCRIPTION
**Pull request by pipcet**

### CODE REVIEW

File: elisp-bytecode.texi [hunk 1]
```diff
@@ -66,6 +67,21 @@ Unbinds special bindings
 @end table
 
 
+@node Constant Instructions
+@section Constant Instructions
+
+The instructions from opcode 192 to 255 encode a constant index from 0
+to 63 as part of the opcode.  For constants outside of that range, see
```
Comment (rocky) [inline elisp-bytecode.texi:74]: Let me see if I understand this correctly. An instruction with opcode 192 means load constant 0 onto the evaluation stack? And an instruction with opcode 193 means load constant 1 onto the evaluation stack?  

And if I want to load constant 100 onto the evaluation stack I would have an instruction with opcode 129 with an operand value of 100?

Comment (pipcet) [reply elisp-bytecode.texi:74]: That's my understanding, yes. The byte sequences would be "192", "193", and "129 100 0"

Comment (pipcet) [reply elisp-bytecode.texi:74]: Okay, I was sufficiently curious to hack up a short elisp routine that displays bytecode and LAP-code side-by-side. It's at https://github.com/pipcet/elisp-lap/commit/f06a9dd83db5fba88f44e1105f3c19cc1ff8a071#diff-69d5d12ecae930bb16ab4c814c7e20f5, and maybe we could use something like that to provide examples of all the opcodes?

Anyway, I've tested with (list 0 1 ... 64), and the relevant bytecode is:

```
    0 192 (byte-constant 0)
    1 193 (byte-constant 1)
    2 194 (byte-constant 2)
[...]
   62 254 (byte-constant 62)
   63 255 (byte-constant 63)
   64 129 (byte-constant 64)
       64
        0
   67 175 (byte-listN . 65)
       65
   69 135 (byte-return . 0)
```
```

Comment (rocky) [reply elisp-bytecode.texi:74]: Again, this is fantastic. I think you should include the .el and make the changes make the doc match. Thanks.

**Pull request merged**